.changeset/thirty-squids-sparkle.md

```diff
@@ -0,0 +1,48 @@
+---
+"@justeattakeaway/pie-icons": minor
+"@justeattakeaway/pie-icons-react": minor
+"@justeattakeaway/pie-icons-vue": minor
+"@justeattakeaway/pie-icons-webc": minor
+---
+
+[Added] - Icons
+
+## Benefits
+- offer-filled
+- offer-filled-large
+
+## Currency
+- krone
+- krone-large
+
+## Functionality
+- archive
+- archive-large
+
+## Pay
+- account-credit-filled
+- account-credit-filled-large
+
+## People
+- user-cancelled
+- user-cancelled-large
+- user-move
+- user-move-large
+
+## Restaurant
+- restaurant-cancelled
+- restaurant-cancelled-large
+
+## Travel & Transport
+- ebike
+- ebike-large
+- ebike-filled
+- ebike-filled-large
+
+[Updated] - Icons
+
+## Pay
+- account-credit
+- account-credit-large
```

Comment:
Are these changes missing?

Reply:
Good catch! Wrong pics name in changeset file, should be `account-credit-filled`